openapi/components/schemas/Rules/Actions/adjust-ready-to-pay.yaml

```
@@ -1,8 +1,14 @@
 allOf:
   - $ref: ../RuleAction.yaml
   - type: object
-    description: Add risk score
+    description: Adjust ready to pay
     properties:
+      pushInstrumentsUp:
+        type: boolean
+        default: false
+        description: |
+          If set as `true`, the payment methods that a customer has a saved payment instrument for will be displayed at
```

I think it should be "active" payment instruments.
```suggestion
          If set as `true`, the payment methods that a customer has active payment instruments for will be displayed at
```